### ISSUE TITLE
bin/build should not log anything until it's sure it's involved

### ISSUE THREAD

**Issue by ericbottard** [cloudfoundry/openjdk-buildpack#4]
Given the way this buildpack works (but actually this applies to others I guess), it should not pollute log and mislead the user in thinking this buildpack is going to be involved when in fact it is not.

Here's an example where no java stuff is going to happen, yet my eye is caught by a red arrow about the openjdk buildpack.

![screen shot 2018-10-18 at 15 11 12](https://user-images.githubusercontent.com/313494/47156852-4532c180-d2e8-11e8-80c5-3e075d571d95.png)

So in other words, I think I'm saying that [this line](https://github.com/cloudfoundry/openjdk-buildpack/blob/8d96cff08eb4a2e3da9f4f5379e999c6c92d0f9b/build/main.go#L35) should be moved inside the `if ok {}` block.

**Comment by nebhale** [comment]
I'd argue against this.  Since the buildpack runs a not insignificant amount of code, potentially occupying a non-trival amount of time, I think it should log its existence.  I also think that it leads to ambiguity if, in the top line of the image it's listed as a pass, but never mentions that it was involved.
